### ISSUE TITLE
Drop brl:created in favour of brl:producedDate or dc:Date

### ISSUE THREAD

**Issue by egli** [daisy/ebraille#198]
From the [Specifications for the Digital Talking Book](http://www.daisy.org/z3986/2005/Z3986-2005.html):

> dc:Date
> Date of publication of the DTB. (Compare dtb:sourceDate and dtb:producedDate.)

> dtb:producedDate
>   Content: Date of first generation of the complete DTB, i.e. Production completion date. (Compare dc:Date.)

**Comment by mattgarrish** [comment]
Do we need both a transcription and publication date or are they always (or almost always) the same thing?

It's likely going to be confusing to people to have a11y:created and dc:date as the metadata properties, which is what we have now.

If we keep both, we should at least give the transcription date a more meaningful name, like dateTranscribed. That would be more in line with the newer dcterms date names.

**Comment by wfree-aph** [comment]
@mattgarrish transcription and publication date can be different. I'm assuming here that publication date is related to the print work? If that's the case, then yes they can be different. If publication date refers to the braille transcription, then it would be the same as the transcription date.

**Comment by mattgarrish** [comment]
> publication date is related to the print work?

No, the publication date would be the date that you publish the braille work.

If we have different metadata for who produces the transcription (dc:producer) and who publishes it (dc:publisher), I assume that both the transcription and publication dates could also differ? I'm just not sure if this difference in dates matters except maybe for in-house purposes.
